.changelog/3903.internal.md

```diff
@@ -0,0 +1,8 @@
+go/cmd/registry: simplify register runtime command
```

Comment:
I don't think this is internal, but rather a "feature" (or even breaking, but we'll want to backport this).

Reply:
I think this should be classified as breaking.